tests/core/io/test_resource.h

```diff
@@ -124,9 +126,11 @@ TEST_CASE("[Resource] Breaking circular references on save") {
 	const String save_path_binary = OS::get_singleton()->get_cache_path().path_join("resource.res");
 	const String save_path_text = OS::get_singleton()->get_cache_path().path_join("resource.tres");
 	ResourceSaver::save(resource_a, save_path_binary);
+	ERR_PRINT_OFF;
```

Comment:
Could you add a short comment explaining why the errors are expected, like you did in the PR?